pallets/ddc-payouts/src/lib.rs

```diff
@@ -202,10 +202,20 @@ pub mod pallet {
 			cluster_id: ClusterId,
```

Comment:
it makes sense to rename the field. I've checkde with DAC and it should be fine. as we have several rewards, it makes sense to name specificly

please rename to ProviderRewarded

Reply:
Updated